diesel/src/lib.rs

```diff
@@ -1,9 +1,13 @@
 //! Diesel is an ORM and query builder designed to reduce the boilerplate for database
 //! interactions. [A getting started guide](http://diesel.rs/guides/getting-started/) can be
 //! found on our website.
-#![deny(warnings, missing_debug_implementations, missing_copy_implementations)]
+#![deny(missing_debug_implementations, missing_copy_implementations)]
 #![cfg_attr(feature = "unstable", feature(specialization))]
 
+#![cfg_attr(feature = "dev", allow(unstable_features))]
+#![cfg_attr(feature = "dev", feature(plugin))]
+#![cfg_attr(feature = "dev", plugin(clippy(conf_file="../clippy.toml")))]
```

Comment:
I would imagine we'd want it whenever possible since it's a lint pass. I
don't feel terribly strongly about it though

On Tue, Apr 26, 2016, 2:36 AM Pascal Hertleif <notifications@github.com>
wrote:

> In diesel/src/lib.rs
> <https://github.com/diesel-rs/diesel/pull/305#discussion_r61047346>:
>
> >  #![cfg_attr(feature = "unstable", feature(specialization))]
> >
> > +#![cfg_attr(feature = "dev", allow(unstable_features))]
> > +#![cfg_attr(feature = "dev", feature(plugin))]
> > +#![cfg_attr(feature = "dev", plugin(clippy(conf_file="../clippy.toml")))]
>
> I created a new feature so we can control if/when CI includes clippy. Do
> you think we should just include it every time unstable is tested?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/diesel-rs/diesel/pull/305/files/9b4d3b77b7c5c046956b10be79757433abbec0b2#r61047346>
>